### PR TITLE
Release v57 for Ubuntu

### PR DESCRIPTION
**Pull request by souldanger**
updated
- linux download page
- news post
- marquee on landing page

other commits were 'tryouts' and testing of forms....

### CODE REVIEW

File: _home/home.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ title: "Home"
 A Browser securing your privacy. That's it.
 ================================  
       
-[Download](downloads/){: .button .download}     
+[Download](downloads/){: .button .big .download}     
 	 
-The iridiumbrowser is based on the Chromium code base. All modifications enhance the privacy of the user and make sure that the latest and best secure technologies are used.
+The IridiumBrowser is based on the Chromium code base. All modifications enhance the privacy of the user and make sure that the latest and best secure technologies are used.
```
Comment (longsleep) [inline _home/home.md]: Is this how Iridium should be written? For me it should be two words.

File: _marquee/marquee.md [hunk 2]
```diff
@@ -3,4 +3,4 @@ layout: default-layout
 title: "Marquee"
 ---
 
-**</>** Issue with Symantec Certs **solved!** check out the latest [News](/index.html#news "latest News") </> 
\ No newline at end of file
+</> **IridiumBrowser** releases **version 57.0 for Ubuntu / Debian / Mint** - read the latest [News](/index.html#news "latest News") </>
```
Comment (longsleep) [inline _marquee/marquee.md]: Same here, do we write Irirdium Browser as one word?

File: _posts/2017-04-04-linux-release-v57.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+title:  "IridiumBrowser releases v57 for Linux"
+description: IridiumBrowser releases v57 for Linux
+date:   2017-04-04 20:00:00 +0200
+author:	by admin
+categories: news
+---
+
+We are happy and a little proud to announce the release of version 57.0 for Ubuntu / Debian / Mint.     
+<!--break-->
+To follow will be releases for the following platforms; **Windows, Mac OS X, openSUSE** and for the first time **Fedora**
```
Comment (longsleep) [inline _posts/2017-04-04-linux-release-v57.md]: "To follow will be ... " does not sound right to me.

File: _posts/2017-04-04-linux-release-v57.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+The latest version of the Chromium code base uses less resources and also includes all the latest security features.
```
Comment (longsleep) [inline _posts/2017-04-04-linux-release-v57.md]: This is a little unspecific to me.

**Comment by longsleep** [comment]
@souldanger i guess you should take a look at Git branching and rebasing, so you can avoid adding test commits to a merge request. Siehe z.B. hier https://git-scm.com/book/de/v1/Git-Branching-Rebasing

In diesem fall, wuerd ich dir vorschlagen den branch interactive auf den letzten bereits gemergten commit zu rebasen und alle test commits wegzulassen.

File: _home/home.md (hunk 1 above)
Comment (souldanger) [inline _home/home.md]: this is how it was written from the beginning and I have continued to use it as one word...

File: _posts/2017-04-04-linux-release-v57.md (hunk 4 above)
Comment (souldanger) [inline _posts/2017-04-04-linux-release-v57.md]: OK - can you be any more specific?

File: _posts/2017-04-04-linux-release-v57.md (hunk 3 above)
Comment (souldanger) [inline _posts/2017-04-04-linux-release-v57.md]: OK will update!

**Pull request closed without merging**